cmd/juju/application/config.go

```diff
@@ -349,7 +349,8 @@ func (c *configCommand) getAllConfig(client ApplicationAPI, ctx *cmd.Context) er
 // valid UTF-8.
 func (c *configCommand) validateValues(ctx *cmd.Context) (map[string]string, error) {
 	settings := map[string]string{}
-	for k, v := range c.configBase.ValsToSet {
+	for k, intfc := range c.configBase.ValsToSet {
+		v := fmt.Sprint(intfc) // `intfc` is generally a string
```

Comment:
what does `intfc` mean here? interface?
maybe keep intfc as v, then change v usage to str or val